### PR TITLE
Fixes #16586 - restart gofer with katello-service

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
set to 30, so that it comes up last.

File: katello/katello-service [hunk 1]
```diff
@@ -19,7 +19,8 @@ SERVICES = {
   'foreman-proxy'            => 20,
   'smart_proxy_dynflow_core' => 20,
   'httpd'                    => 30,
-  'foreman-tasks'            => 30
+  'foreman-tasks'            => 30,
+  'goferd'		     => 30
```
Comment (daviddavis) [inline katello/katello-service]: nitpick: mind aligning the hash rocket here with the other lines?

**Review by daviddavis — CHANGES_REQUESTED** [review]
Looks good to me (aside from one small comment).

**Comment by chris1984** [comment]
@daviddavis thanks the review, corrections made with a new commit

**Review by daviddavis — APPROVED** [review]
ACK. I don't have merge access to this repo tho.

**Comment by stbenjam** [comment]
Thanks @chris1984 , @daviddavis

**Pull request merged**